### ISSUE TITLE
SikuliX looking for Tesseract in wrong directory for Mac M1

### ISSUE THREAD

**Issue by jonastoft** [RaiMan/SikuliX1#453]
(PS: Really appreciate the great work, this is such fantastic software!)

On Macbook Air M1 Sikuli looks for Tesseract in _/usr/local/lib_, while it defaults to being installed in _/opt/homebrew_ by Homebrew

Error thrown: **OCR: validate: libtesseract.dylib not in /usr/local/lib**

**Comment by RaiMan** [comment]
Thanks for feedback.

Seems to be special for macOS on M1.

On my macOS 11.4 on Intel it symlinks into /usr/local/lib and the Homebrew stuff is in /usr/local/Cellar.

As a workaround you have to create a symlink for libtesseract.dylib into /usr/local/lib.

Hopefully after Juli 1st I will have my own M1 machine and will then be able to do some testing.

**Comment by jonastoft** [comment]
Thanks RaiMan :)
I tried adding a symlink, but now it just gives the more standard error:

```
[error] see: https://github.com/RaiMan/SikuliX1/wiki/macOS-Linux:-Support-libraries-for-Tess4J-Tesseract-4-OCR
[error] Save your work, correct the problem and restart the IDE!
[error] script [ test ] stopped with error in line 1
[error] Native library (darwin/libtesseract.dylib) not found in resource path ([file ( "Sikuli jar path"]) )
[error] --- Traceback --- error source first
line: module ( function ) statement 
1: main (  <module> )     print(Region.text())
[error] --- Traceback --- end --------------
```

So looks like there's something more going on, unfortunately

**Comment by RaiMan** [comment]
Supposing you are using 2.0.5 on Java 11+

Internally we check wether
`System.getProperty("os.name").toLowerCase()` 
starts with `mac`.

If true, we check wether there is a `libtesseract.dylib` available and if true set the JNA path to `/usr/local/lib`.

Just checked: this works for me on macOS 11.4 Intel with Java 16

**Comment by RaiMan** [comment]
basic fix in 2.0.6-M1: if Tesseract is installed via standard M1-Homebrew, the the dylib is found in /opt/homebrew/lib
